data/formList.js

```diff
@@ -0,0 +1,24 @@
+export default [
+  { text: 'Adult Referral', value: '/form/adult-referral/client-details' },
```

Comment:
Not sure this one should be here 🤔 
At the moment it's the "create new user form"